karma.conf.js

```diff
@@ -0,0 +1,50 @@
+// Karma configuration
```

Comment:
any way to suppress this error?

![screen shot 2017-04-18 at 12 57 23 am](https://cloud.githubusercontent.com/assets/419567/25115207/98185ec6-23d2-11e7-8669-2f23d859a16f.png)